shared-bin/src/logger.rs

```diff
@@ -113,9 +122,23 @@ pub fn init(
         None
     };
 
-    Registry::default()
-        .with(Layer::and_then(telemetry, fmt_layer).with_filter(filter))
+    let mut error_only = vec![
+        "Broadcast offers to taker",
+        "Send message every interval (quiet)",
+    ];
+
+    if !noisy_libp2p_spans {
+        error_only.extend(["Ping peer", "Broadcast offers to taker (libp2p)"]);
+    }
```

Comment:
❓ Do these strings need to match the events/spans they correspond to for this to work? I guess so. This is a bit brittle as soon as we make any changes to the originals :(